h/views/api.py

```diff
@@ -304,6 +292,29 @@ def _json_payload(request):
         raise PayloadError()
 
 
+def _call_search_lib(request, json_presenter=None):
```

Comment:
In my personal opinion, this would be a nicer being called `_execute_search`, `_search_annotations`, or something along those lines. But that's my personal preference, so feel free to ignore me.